table.csv

```diff
@@ -501,7 +501,7 @@ xxh3-64,                        hash,           0xb3e3,         draft,      Extr
 xxh3-128,                       hash,           0xb3e4,         draft,      Extremely fast non-cryptographic hash algorithm
 poseidon-bls12_381-a2-fc1,      multihash,      0xb401,         permanent,  Poseidon using BLS12-381 and arity of 2 with Filecoin parameters
 poseidon-bls12_381-a2-fc1-sc,   multihash,      0xb402,         draft,      Poseidon using BLS12-381 and arity of 2 with Filecoin parameters - high-security variant
-urdca-2015-canon,               ipld,           0xb403,         draft,      The result of canonicalizing an input according to URDCA-2015 and then expressing its hash value as a multihash value.
+rdfc-1-0,                       ipld,           0xb403,         draft,      The result of canonicalizing an input according to RDFC-1.0 and then expressing its hash value as a multihash value.
```

Comment:
```suggestion
rdfc-1,                       ipld,           0xb403,         draft,      The result of canonicalizing an input according to RDFC-1.0 and then expressing its hash value as a multihash value.
```

I expect that rdfc-1 is just fine, and a point-release would be backwards-compatible. If we need to, we can do rdfc-1.1/rdfc-1-1 in the future. Suggest just keeping it to a major number for now.

Reply:
I think `rdfc-1` is fine here. Any changes to the algorithm wouldn't change outputs (without changing a major version) but rather just be some kind of spec fixes for minor versions, so we only need one codec per major version, IMO.

Reply:
Agreed re `rdfc-1`

Reply:
@peacekeeper, if you're able to apply this suggestion to your PR I think it's good to merge.